content/Link Tracking with StimulusReflex - Part III.md

```diff
@@ -0,0 +1,156 @@
+# Link Tracking with StimulusReflex - Part II
+
+In this last part of this series, we'll use what [we built before](TODO: Add link to second part) to actually start tracking views. To do so, we'll use [ActionCable](https://guides.rubyonrails.org/action_cable_overview.html) and [CableReady](https://github.com/hopsoft/cable_ready) to broadcast and make changes in our frontend.
+
+![Header image](images/stimulus_reflex.jpeg)
+
+### Redirecting users
+
+Let's start with the simple stuff, when a user follows one of our shortened links, they expect to be redirected to the original URL. To do this, we'll create a new controller and add the corresponding route.
+
+```ruby
+# app/controllers/redirections_controller.rb
+
+class RedirectionsController < ApplicationController
+  def index
+  end
+end
+```
+
+```ruby
+# config/routes.rb
+
+Rails.application.routes.draw do
+  # ...
+  get '/:path', to: 'redirections#index'
```

Comment:
maybe clarify that the route MUST be at the bottom of the block, otherwise it'd override other routes